goal_src/engine/ambient/mood-tables.gc

```diff
@@ -5,3 +5,4106 @@
 ;; name in dgo: mood-tables
 ;; dgos: GAME, ENGINE
 
+;; definition for function make-light-kit
+(defun
+  make-light-kit
+  ((arg0 light-group) (arg1 float) (arg2 float) (arg3 float) (arg4 float))
```

Comment:
I like to put all these on the same line:
`(defun make-light-kit ((arg0 light-group) (arg1 float) (arg2 float) (arg3 float) (arg4 float))`
Not a huge deal, but having the name on the same line as `defun` makes it easier to search for `defun make-light-kit`.